### PR TITLE
Download

### PR DESCRIPTION
**Pull request by NeoTheThird**
Implement downloads

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,22 @@ Access a system-image server http endpoint
...
-const systemImageClient = require("system-image-node-module").Client;
-const sic = new systemImageClient();
-sic.getDeviceChannels("bacon").then((channels) => console.log(channels));
+
+const systemImageClient = require("./src/module.js").Client;
+const systemImage = new systemImageClient();
+
+systemImage.getDeviceChannels("bacon").then((channels) => console.log(channels));
+
+var progress = (progress, speed) => {
```
Comment (mariogrip) [inline README.md]: small thing, but it's better to make functions that wont change const

**Comment by NeoTheThird** [comment]
Ignore coveralls, we'll do tests later.

**Review by mariogrip — APPROVED** [review]
Small comment, but otherwise lgtm :+1:

**Comment by NeoTheThird** [comment]
Okay, thanks!

**Pull request merged**